server/bin/pbench-sync-satellite.sh

```diff
@@ -3,92 +3,87 @@
 # Simulate what pbench-move-results does for tar balls found on a given
 # satellite (remote) pbench server.
 
-# pbench-move-results copies tarballs to the server in the reception
-# areas (fs-version-001 and -002 versions), and then the respective
-# shims set about moving them into place for pbench-dispatch to move
-# through the various processing stages.
+# pbench-move-results copies tarballs to the server in the reception areas via
+# SSH (scp really; fs-version-002 version only considered), and then dispatch
+# sets about moving them through the various processing stages.
 
-# This script simulates what pbench-move-results does with the
-# tarballs that it copies from a satellite server. It runs as a cron
-# job once a minute.
+# This script simulates what pbench-move-results does with the tarballs that it
+# copies from a satellite server. It runs as a cron job once a minute.
 
-# Assumption: this script is running as user "pbench" on the master
-# server and can ssh as user "pbench" to the given satellite server,
-# without a password.
+# Assumption: this script is running as user "pbench" on the main server and can
+# `ssh` as user "pbench" to the given satellite server, without a password.
 
-# load common things
-case $# in
+# Load common things
+case ${#} in
     1)
         :
         ;;
     *)
-        echo "Usage: $PROG <satellite-config>" >&2
+        echo "Usage: ${PROG} <satellite-config>" >&2
         exit 1
         ;;
 esac
-satellite_config=$1
+satellite_config=${1}
 shift 1
-. $dir/pbench-base.sh
+. ${dir}/pbench-base.sh
 
 test -d "${ARCHIVE}" || doexit "Bad ARCHIVE=${ARCHIVE}"
 
 remote_prefix=$(getconf.py satellite-prefix ${satellite_config})
 remote_host=$(getconf.py satellite-host ${satellite_config})
 
-tmp=$(get-tempdir-name $PROG)
-unpack=$tmp/unpack.$remote_prefix
-mkdir -p $unpack || doexit "Failed to create $unpack"
+tmp=$(get-tempdir-name ${PROG})
+unpack=${tmp}/unpack.${remote_prefix}
+mkdir -p ${unpack} || doexit "Failed to create ${unpack}"
 
-# Be sure $logdir is defined before setting up the trap below.
-logdir_for_remote=$LOGSDIR/$PROG/$remote_prefix
-logdir=$logdir_for_remote/$TS
+# Be sure ${logdir} is defined before setting up the `trap` below.
```

Comment:
Not relevant to your changes, so I'll pre-resolve; but this comment looks odd here since the commands in this "block" don't even reference `${logdir}`. Also, there's already a comment before the `trap`. If this is intended to be an "early warning" it should be moved to the top, but otherwise it could probably just be dropped.